README.md

```diff
@@ -71,7 +71,7 @@ practices that anyone working with the SDK needs to be aware of and uphold:
 
   * The view's CSS file MUST have the same name (e.g. view/rooms/MessageTile.css).
     CSS for matrix-react-sdk currently resides in
-    https://github.com/vector-im/element-web/tree/master/src/skins/vector/css/matrix-react-sdk.
+    https://github.com/matrix-org/matrix-react-sdk/tree/develop/res/css.
```

Comment:
I think this should link to `master` branch instead of `develop`.

Reply:
> I think this should link to `master` branch instead of `develop`.

Yaa sure, I will change the branch.